src/ray/protobuf/new_monitor.proto

```diff
@@ -0,0 +1,201 @@
+// ============= Scheduling Status Primitives ====================
+//
+// Following fields represents the Ray state autoscaler interested
+// in.
+
+// Represents anti affinity constraints for a resource requirement.
```

Comment:
How is this represented: (anti-)affinity with nodes who has label k1 with value v1 or v2?

```
# Try to schedule to the node with A100/P100 graphics card. If not, schedule to other nodes.
actor_1 = Actor.options(
        scheduling_strategy=node_affinity(label_in("gpu_type", ["A100", "P100"], is_soft=true))
    ).remote()
```